packages/adminpanel-v2/src/lib/mock/customers.ts

```diff
@@ -30,6 +30,7 @@ export const Projects: TProject[] = [
 		projectNumber: 1,
 		startDate: new Date('December 17, 1995 03:24:00'),
 		customer: 1,
+		changedDate: new Date('December 17, 1995 03:24:00').toLocaleString('en-GB', { timeZone: 'UTC' }).split(',')[0],
```

Comment:
Put string formatting in a function

Reply:
Added a formatting function in the store. Didn't change anything in the mock data, as this is just temporary test data.